go.sum

```diff
@@ -10,40 +10,33 @@ cloud.google.com/go v0.52.0/go.mod h1:pXajvRH/6o3+F9jDHZWQ5PbGhn+o8w9qiu/CffaVdO
 cloud.google.com/go v0.53.0/go.mod h1:fp/UouUEsRkN6ryDKNW/Upv/JBKnv6WDthjR6+vze6M=
 cloud.google.com/go v0.54.0/go.mod h1:1rq2OEkV3YMf6n/9ZvGWI3GWw0VoqH/1x2nd8Is/bPc=
 cloud.google.com/go v0.56.0/go.mod h1:jr7tqZxxKOVYizybht9+26Z/gUq7tiRzu+ACVAMbKVk=
```

Comment:
Should I revert the go sum changes? Not sure why it changed, looks like the hashes changed

Reply:
Yeah, you should revert them as they shouldn't make too much of a difference.

Reply:
+1 for revert

Reply:
I attempted to resolve this. I think there is one more problem with it regarding whitespace but the hash is the same. any idea on how to fix that?